VCU/Phantom/Drivers/RTOS/phantom_freertos.h

```diff
@@ -40,6 +40,10 @@
 /* This timer is used to debounce the interrupts for the RTDS and SDC signals */
 void Timer_300ms(TimerHandle_t xTimers);
 void Timer_2s(TimerHandle_t xTimers);
+void APPS1_SEVERE_RANGE_FAULT_CALLBACK(TimerHandle_t xTimers);      // prototype for APPS1 severe range fault software timer callback - jaypacamarra
+void APPS2_SEVERE_RANGE_FAULT_CALLBACK(TimerHandle_t xTimers);      // prototype for APPS2 severe range fault software timer callback - jaypacamarra
+void BSE_SEVERE_RANGE_FAULT_CALLBACK(TimerHandle_t xTimers);        // prototype for BSE severe range fault software timer callback - jaypacamarra
+void FP_DIFF_SEVERE_FAULT_CALLBACK(TimerHandle_t xTimers);          // prototype for FP difference by 10% fault software timer callback - jaypacamarra
```

Comment:
You have 6 timer callbacks, change NUMBER_OF_TIMERS at line 38 to 6.

Reply:
Oops. Fixed that.